DESCRIPTION

```diff
@@ -1,44 +1,47 @@
 Package: allodb
-Title: An R database for biomass estimation at globally distributed extratropical forest plots.
-Version: 1.0
+Title: An R Database For Biomass Estimation At Globally Distributed Extratropical Forest Plots
```

Comment:
I fixed the case and removed the period, but I believe one other problem remains -- the title may need to be 80 characters or less.

Reply:
You can change it to: An R Database For Biomass Estimation At Extratropical Forest Plots

(66 characters)

Reply:
ok changed it